sdk/storage/azure-storage-file-share/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 ## 12.13.0-beta.2 (Unreleased)
 
 ### Features Added
+- Added support for the 2021-06-08 service version.
```

Comment:
It looks like you are merging to main here, so this comment isn't relevant.

In the future, I would recommend creating and an STG XX (XX being 83, 84, etc.) base branch, which adds the new service version, and any other work that will be duplicated with every STG XX PR, and then branching off the STG XX base branch for each feature.

I then update the changelogs for STG XX as a separate PR, right before merging to main.